utils/issue-format-bot/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "issue-format-bot",
+  "version": "1.0.0",
+  "description": "GitHub bot to parse ruleset issues",
+  "author": "AJ Jordan <alex@strugee.net>",
+  "main": "index.js",
+  "license": "AGPL-3.0+",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/EFForg/HTTPS-Everywhere.git"
+  },
+  "scripts": {
+    "lint": "eslint index.js test lib",
+    "start": "probot run ./index.js",
```

Comment:
I'll probably edit this once we merge it to add the `-t` flag for a custom subdomain.  I'll be running this inside Docker rather than Heroku, so this makes us able to specify a URL manually.